### ISSUE TITLE
Hosted file moved? 

### ISSUE THREAD

**Issue by yonghaoy** [igvteam/igv.js#1579]
Hello, IGV.js is broken recently because `https://igv.genepattern.org/genomes/seq/hg38/hg38.fa.fai` is blocked by our Content Security Policy.
It is expected because we have Content Security Policy allowlist and that allowlist does not contain that url. 
IGV urls in our current allowlist are
```
    "https://s3.amazonaws.com/igv.broadinstitute.org/",
    "https://s3.amazonaws.com/igv.org.genomes/",
    "https://portals.broadinstitute.org/webservices/igv/",
    "https://igv.org/genomes/",
```

I am wondering if igv.js recently move their hosted files to igv.genepattern.org? 
I saw @jrobinso mentioned the moving here: https://github.com/igvteam/igv.js/issues/1570#issuecomment-1357951675
but I am not sure if that is the same thing. 

Thanks!

**Comment by jrobinso** [comment]
Yes, we will be slowing moving all our data to that host.    If you want to absolutely protect yourself against data moves you could host genomes on your own server,  but we don't move data often.

**Comment by yonghaoy** [comment]
Thanks @jrobinso 
To confirm, is hg38.fa.fai moved to  igv.genepattern.org already? 
I am investigating why igv is broken recently, and want to confirm adding https://igv.genepattern.org to our CSP allowlist can fix this issue. 
Thanks
Yonghao

**Comment by jrobinso** [comment]
Yes.   But again,  if this is an issue for your organization you might consider hosting the files you need within your organization locally,  we host data for IGV as a convenience but it is not required you use our hosted data.  Costs are becoming an issue and files could move again in the future.   Some instructions are here: https://github.com/igvteam/igv/wiki/Hosting-Genomes

Another important host, and I'm surprised IGV works at all without this one whitelisted is

https://data.broadinstitute.org/igvdata

Also, the following might well be used

https://igv-genepattern-org.s3.amazonaws.com

**Comment by yonghaoy** [comment]
Thanks for getting this back. 
I am not familiar with IGV and how to use it. We are developing tools for scientists who are heavily using IGV. 
The python code snippet that stop working(and I don't know how/where to set host data) is 
```
import igv

b = igv.Browser({"genome": "hg38"})
b.load_track(
    {
        "name": "wgs_1000004",
        "url": "wgs_1000004.cram",
        "format": "cram",
        "type": "alignment",
        "indexURL": "wgs_1000004.cram.crai",
        "indexed": True
    })

b.show()
``` 

> RE: https://data.broadinstitute.org/igvdata

The server(https://app.terra.bio/) we are developing is hosted by Broad. And I think all Broad hostname is allowed. 

RE: 

> https://igv-genepattern-org.s3.amazonaws.com/ Thanks! I will also add that into our allowlist...

**Comment by jrobinso** [comment]
Ahh, you are actually using https://github.com/igvteam/igv-notebook then.   OK, sorry, my suggestion to consider hosting data on your own servers still apply, however the instructions I pointed to was for IGV desktop. 

What version of igv-notebook are you using?    The most recent is 0.4.4,  although for all practical purposes this is ready to release as 1.0.0. https://pypi.org/project/igv-notebook/

The configuration looks suspect,  in particular the url and indexURL are not qualifed.   I'm not sure how this is working, but if it was working before then the host name change might be the problem.

I'm going to transfer this issue to igv-notebook.
